enterprise/users/src/main/java/io/crate/operation/user/UserManagerService.java

```diff
@@ -158,6 +161,18 @@ public UserManagerService(TransportCreateUserAction transportCreateUserAction,
         return listener;
     }
 
+    @Override
+    public CompletableFuture<Long> alterUser(String userName, @Nullable SecureHash secureHash) {
+        FutureActionListener<WriteUserResponse, Long> listener = new FutureActionListener<>(r -> {
+            if (r.doesUserExist() == false) {
+                throw new UserUnknownException(userName);
```

Comment:
could this be used by an unprivileged user to figure out other user names?

Reply:
Not sure if I get you right here. However, a brute-force attack wouldn't be sufficient here since the an unprivileged user would rise an `UnauthorizedException` earlier than a `UserUnknownException`.

```sql
foo=> alter user mibe set password = null;
ERROR:  UnauthorizedException: User "foo" is not authorized to execute statement
```